src/frontend/index.tsx

```diff
@@ -134,6 +135,20 @@ window.setTheme = async (themeClass: string) => {
   }
 
   document.body.className = themeClass
+
+  if (navigator['windowControlsOverlay']?.visible) {
+    const titlebarOverlay = Object.fromEntries(
+      ['height', 'color', 'symbol-color']
+        .map((item) => [
+          camelCase(item),
```

Comment:
I think we are adding a new library just to call this camelCase here and it's only for the `system-color` string

I'd say to just hardcode the value we want

I think it's overkill to add 5 dependencies for this one place (camelCase adds other `case`-related dependencies)

Reply:
also, it looks like we already have another `camelcase` dependency (note this one is `camelcase` not `camelCase`) from what I see in the yarn.lock diff

Reply:
Agreed. Didn't notice the existing camelcase package, but it looks like that's only depended on by development packages so I'm not sure it can be relied on to be present in a production build. I'll just hardcode this instead.